jetty-websocket/websocket-javax-common/src/main/java/org/eclipse/jetty/websocket/javax/common/decoders/RegisteredDecoder.java

```diff
@@ -74,6 +78,23 @@ public boolean isType(Class<?> type)
         return (T)instance;
     }
 
+    public void destroyInstance()
```

Comment:
We have an interface `Destroyable`, any reason to not use it?

Reply:
Is the `Destroyable` interface exposed to the WebApp?

Reply:
> Is the Destroyable interface exposed to the WebApp?

I don't think it needs to be. `RegisteredDecoder` is an internal implementation anyway not something used by the WebApp.

Reply:
I don't think there is any benefit from implementing `Destroyable` though, we don't really need the JMX operations for destroy on this.

This usage doesn't really line up with the description of `Destroyable` from the javadoc:
> Typically a Destroyable is a {@link LifeCycle} component that can hold onto
>  resources over multiple start/stop cycles. A call to destroy will release all
>  resources and will prevent any further start/stop cycles from being successful.

Reply:
Fair enough.